EmployeeLib/LSPModel/ISuperVisor.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace EmployeeLib.LSPModel
+{
+    public interface ISuperVisor
```

Comment:
the supervisor naming is a bit harder to follow when it's the same as manager. consider to rename this or the manager to make it more consistent